docs/reference/buildx_build.md

```diff
@@ -217,27 +224,36 @@ $ docker buildx build --cache-from=user/app:cache .
 $ docker buildx build --cache-from=user/app .
 $ docker buildx build --cache-from=type=registry,ref=user/app .
 $ docker buildx build --cache-from=type=local,src=path/to/cache .
+$ docker buildx build --cache-from=type=gha .
 ```
 
+More info about cache exporters and available attributes: https://github.com/moby/buildkit#export-cache
```

Comment:
We should probably include a table somewhere in a Markdown document inside `docs/`  (perhaps_ move_ the information from the readme to that page), otherwise the docs at docs.docker.com will link to the GitHub page (instead of showing the information in the docs)